### PR TITLE
[DNM] CEF 75 update

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Not meant for Revit 2021 release, trial update to CEF75.1.143. I was able to resolve initialization error and dependencies error by coping everything in the CEF folder over to Dynamo folder, seems new dependencies introduced. But still got a blank screen so will need to put more time in it looking at API usage maybe.

image

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

**Comment by QilongTang** [comment]
Ignore the Appveyor failures because they are for library test project build failures

**Comment by mjkkirschner** [comment]
@QilongTang so.... I take it this is not yet working :wink?

File: src/LibraryViewExtension/LibraryViewExtension.csproj [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-  <Import Project="..\packages\CefSharp.Wpf.65.0.1\build\CefSharp.Wpf.props" Condition="Exists('..\packages\CefSharp.Wpf.65.0.1\build\CefSharp.Wpf.props')" />
-  <Import Project="..\packages\CefSharp.Common.65.0.1\build\CefSharp.Common.props" Condition="Exists('..\packages\CefSharp.Common.65.0.1\build\CefSharp.Common.props')" />
-  <Import Project="..\packages\cef.redist.x86.3.3325.1758\build\cef.redist.x86.props" Condition="Exists('..\packages\cef.redist.x86.3.3325.1758\build\cef.redist.x86.props')" />
-  <Import Project="..\packages\cef.redist.x64.3.3325.1758\build\cef.redist.x64.props" Condition="Exists('..\packages\cef.redist.x64.3.3325.1758\build\cef.redist.x64.props')" />
+  <Import Project="..\packages\CefSharp.Wpf.75.1.143\build\CefSharp.Wpf.props" Condition="Exists('..\packages\CefSharp.Wpf.75.1.143\build\CefSharp.Wpf.props')" />
+  <Import Project="..\packages\CefSharp.Common.75.1.143\build\CefSharp.Common.props" Condition="Exists('..\packages\CefSharp.Common.75.1.143\build\CefSharp.Common.props')" />
+  <Import Project="..\packages\cef.redist.x86.77.1.18\build\cef.redist.x86.props" Condition="Exists('..\packages\cef.redist.x86.77.1.18\build\cef.redist.x86.props')" />
```
Comment (mjkkirschner) [inline src/LibraryViewExtension/LibraryViewExtension.csproj]: @QilongTang don't reference version 77 - the CEF version should match the CEFSharp version... I am pretty sure of this.

**Comment by mjkkirschner** [comment]
@QilongTang - you might try altering our startup CefSharp settings to:
https://github.com/cefsharp/CefSharp/blob/cefsharp/75/CefSharp.Wpf/CefSettings.cs#L29

**Comment by QilongTang** [comment]
@mjkkirschner Hi I dont see it under CEF Settings here, anything I missed?
image

**Comment by QilongTang** [comment]
After some searching, seems we should use a different approach now, see https://stackoverflow.com/questions/59124372/cefsharp-how-can-i-access-onresourceresponse-from-build-75-1-143

**Comment by QilongTang** [comment]
closing for now

**Pull request closed without merging**